pennylane/transforms/control.py

```diff
@@ -134,18 +171,23 @@ def adjoint(self):
                 # Execute all ops adjointed.
                 adjoint(requeue_ops_in_tape)(self.subtape)
 
-        return ControlledOperation(new_tape, self.control_wires)
+        return ControlledOperation(new_tape, self.control_wires, control_values=self.control_values)
 
     def _controlled(self, wires):
-        ControlledOperation(tape=self.subtape, control_wires=Wires(wires) + self.control_wires)
+        ControlledOperation(
+            tape=self.subtape,
+            control_wires=Wires(wires) + self.control_wires,
+            control_values=self.control_values,

```

Comment:
```suggestion
            control_values=self.control_values+[1],
```

Reply:
Actually, I think _controlled needs to handle more than one additional wire. I will push a modification that accounts for this fact.